templates/user/privacy.html

```diff
@@ -1,41 +1,112 @@
 {% extends "layout.html" %}
 
+{% block extra_css %}
+<link rel=stylesheet href={{ url_for('static', filename='css/privacy.css') }}>
+{% endblock %}
+
 {% block main %}
-<style>
-table {
-  margin: 0 auto;
-  border-collapse: collapse;
-}
-tr:not(:first-child,:last-child) {
-  border-bottom: 1px solid #ccc;
-}
-td {
-  padding: 1rem;
-}
-.delete {
-  background-color: #fee7e7;
-  color: #a24343;
-  border-color: #dab2b2;
+<section>
+  <article>
+    <h2>Community mode</h2>
+    <form>
+      <input type=checkbox name=community {% if user.allow_community %}checked{% endif %} id=community>
+      <label for=community title="{% if user.allow_community %}Disable{% else %}Enable{% endif %} community profile">
+        Click to {% if user.allow_community %}disable{% else %}enable{% endif %}
+      </label>
+    </form>
+    <aside>
+      {% if user.allow_community %}
+      <p>
+        Disable community mode hidden your public data as publication, localisation and account.
+      </p>
+      {% else %}
+      <p>
+        Enable community mode allow you to share your articles, localisation and account.
```

Comment:
Enabling community mode allows you to publish your articles, to share your location, and visitors to read your profile.